src/cls/IPM/Main.cls

```diff
@@ -2803,9 +2811,12 @@ ZPM(pArgs...)
 		Set found = 0	
 		While rs.%Next() {
 			Set $Namespace = $Zstrip(rs.%Get("Nsp"), "<>WC")
-			If $System.CLS.IsMthd("%IPM.Main", "Shell") {
+			// Some I4H containers come with %IPM.Main but the "version" command doesn't work ?!
```

Comment:
This would probably be those with IPM mapped from HSLIB/HSCUSTOM, on older versions?